dev/com.ibm.ws.app.manager.springboot/src/com/ibm/ws/app/manager/springboot/util/SpringBootThinUtil.java

```diff
@@ -1347,6 +1442,7 @@ public static Set<String> getStarterArtifactIds(String starter) {
                                                                                              "io.undertow:undertow-websockets-jsr:jar:2.3.10.Final:compile",
                                                                                              "jakarta.servlet:jakarta.servlet-api:jar:6.0.0:compile");
         private final static List<String> mvnSpringBoot32UndertowStarterDeps = Arrays.asList(
+
```

Comment:
Did you intend to move `org.springframework.boot:spring-boot-starter-undertow:jar:3.2.1:compile` to this line?

Reply:
I think i missed moving the starter entry to the top in this list. Thank, Tom!